test/e2e/framework/create.go

```diff
@@ -552,6 +554,27 @@ func (*storageClassFactory) Create(f *Framework, i interface{}) (func() error, e
 	}, nil
 }
 
+type csiDriverFactory struct{}
+
+func (f *csiDriverFactory) New() runtime.Object {
+	return &storagev1beta1.CSIDriver{}
+}
+
+func (*csiDriverFactory) Create(f *Framework, i interface{}) (func() error, error) {
```

Comment:
Where are these used and what do they do?

Reply:
I thought it is needed for e2e tests as I saw this is done for other objects, but maybe not needed.  I'll double check.

Reply:
It is used to load manifests: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/create.go#L61 to load manifests. Then it is called here: 
 https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/csi.go#L155

Reply:
The e2e tests used to install the CSIDriver CRD manually. But we changed them now to use cluster-driver-registrar.  So I think we can remove this.

Reply:
Ok. Removed it.